tools/pd-ctl/pdctl/command/region_command.go

```diff
@@ -48,6 +48,7 @@ var (
 	regionsKeyspacePrefix   = "pd/api/v1/regions/keyspace"
 	regionIDPrefix          = "pd/api/v1/region/id"
 	regionKeyPrefix         = "pd/api/v1/region/key"
+	regionRulePrefix        = "pd/api/v1/config/rules/region"
```

Comment:
Does it belong to region or config?

Reply:
I'm not sure too. How about `config placement-rules region <region-id>` or `region rules <region-id>`